### DISCUSSION TITLE
SAM2 and Interactive segmentation

### DISCUSSION THREAD

**Discussion by aiEnthusiast2024** [opengeos/segment-geospatial#378] [Q&A]
Hello! 
Not sure if anyone else has run into this issue before, but in the interactive segmentation in the jupyter notebook, once I select set of foreground and background points, and click on Segment, seems like there is some processing that happens in the background, but nothing happens after that. No output or segmentation mask shows up on the sam.show_map() window. Any help is appreciated.

Also - in this notebook 
https://colab.research.google.com/github/opengeos/segment-geospatial/blob/main/docs/examples/sam2_point_prompts.ipynb

running the cell sam.predict_by_points gives the following error -> "SamGeo2 object has no attribute 'predict_by_points' " I must be missing something?

**Comment by giswqs** [comment]
Some recent dependency update might have caused this. I need to look into it.
